lib/elements/rebar.rb

```diff
@@ -0,0 +1,66 @@
+require './db/base'
+require './lib/materials/steel'
+
+module Elements
+  class Rebar < Materials::Steel
+    MAX_YIELD_STRESS = 550.to_f
+
+    attr_reader :diameter, :yield_stress, :number, :hooks_class, :elastic_module, :start_hook, :end_hook
```

Comment:
what is `hooks_class`? I don't see it being used anywhere
`elastic_module` and `yield_stress` shouldn't be attributes of rebar, but of the rebar's material

Reply:
that's true!

```ruby
attr_reader :diameter, :number, :start_hook, :end_hook, :material
```